app/views/static/default_landing/partials/_action_button.html.erb

```
@@ -0,0 +1,5 @@
+<% raise 'Could not find action button content' if (local_assigns['url'] && local_assigns['text']).nil? %>
```

The `.nil?` check is quite specific. It evaluates if it's not set at all, but what about if it's set with an empty string? Is this a state that we want to raise an error on?

```
irb(main):001:0> true.nil?
=> false
irb(main):002:0> false.nil?
=> false
irb(main):003:0> nil.nil?
=> true
irb(main):004:0> "".nil?
=> false
irb(main):005:0> (true && nil).nil?
=> true
irb(main):006:0> (true && "").nil?
=> false
```